src/test/java/org/jabref/logic/util/io/RegExpBasedFileFinderTests.java

```diff
@@ -143,34 +156,4 @@ void testFindFileNonRecursive() throws Exception {
         // then
         assertTrue(result.isEmpty());
     }
-
-    @Test
-    void testExpandBrackets() {
```

Comment:
We should try to keep these tests. We are discussing at https://github.com/JabRef/jabref/issues/4342#issuecomment-762515082.

Reply:
I'll take another look at it but I am 90% sure they only duplicate the `expandBrackets` of `BracketedPattern`. Perhaps they can be moved there instead.

Reply:
Wait a sec. Just to be perfectly clear. I have not removed this functionality, I am using the equivalent code in `BracketedPattern`.

Reply:
Due to other bugs (mostly with parsing the regex modifier), I am slowly trying to move all parsing of bracketed patterns to `BracketedPattern.java`.

Reply:
It's good to move extensive testing indeed to `BracketedPattern`. However, the should at least one test here to verify that the file finder expands pattern as expected (otherwise you could remove the internal call to bracketedpattern and still all tests would pass).

Reply:
> However, the should at least one test here to verify that the file finder expands pattern as expected

I intended https://github.com/JabRef/jabref/pull/7338/files/4c2f2c3cba5c8636cd8854a6fc3aa51beb0053a2#r560415435 to cover that scenario. Admittedly, it might make sense to cover more complex cases than this.

Reply:
There is no need to save disk space or test class length. May I ask to either keep these tests (they should be green before and after your fix) or move them to `BracketedPattern`? - Even if they are pretty similar, they form a good regression test to ensure that future code "optimizations" keep the old behavior.

I also think that these tests can also serve as "documentation" of the class `RegExpBasedFileFinder`, even if that class relies to other classes, where documentation is found. I really like that the public interface of the class is tested.

Reply:
I'll add them to the tests in `BracketedPattern` with a hint/comment that they are from `RegExpBasedFileFinder`.

Reply:
The new tests are way better since they actually test the indented behavior (files are found) instead of an implementation detail (expandBrackets method).